### PR TITLE
feat: add all-set message, with Actions link

### PR DESCRIPTION
**Pull request by Wattenberger**

### CODE REVIEW

File: src/flatConfigEditor.ts [hunk 1]
```diff
@@ -164,12 +164,29 @@ export class FlatConfigEditor implements vscode.CustomTextEditorProvider {
...
     const document = await vscode.workspace.openTextDocument(flatFileUri)
     const rawFlatYaml = document.getText()
-    const parsedConfig = parse(rawFlatYaml)
 
     const dirName = workspaceRootUri.path.substring(
       workspaceRootUri.path.lastIndexOf('/') + 1
     )
 
+    let gitRepo = ''
+    const gitConfigFileUri = vscode.Uri.joinPath(
```
Comment (mattrothenberg) [inline src/flatConfigEditor.ts:173]: Any reason you're not using the Git API class that we previously extracted for this purpose?

Comment (Wattenberger) [reply src/flatConfigEditor.ts:173]: yess you're right! I didn't want to pull in an extra lib when it wasn't necessary but I had totally forgotten it was already there. will update!

Comment (mattrothenberg) [reply src/flatConfigEditor.ts:173]: @Wattenberger No need to change, just wanted to understand your thought process here!

Comment (Wattenberger) [reply src/flatConfigEditor.ts]: nah updated! that was a great call

**Review by mattrothenberg — APPROVED** [review]

**Pull request merged**